src/web/layout/layout.md

```diff
@@ -0,0 +1,379 @@
+---
+title: "CSS: Layout"
+level: 4
+language: nb-NO
+---
+
+# Introduksjon {.intro}
+
+Målet med oppgaven er å lære hvordan man lager en nettside med en meny og et innholdsfelt.  
+
+I denne oppgaven forventer vi at du har vært gjennom HTML- og CSS-oppgavene så langt og/eller er kjent med `<div>`- og HTML5-taggene. 
+
+
+![resultat](ressurser/screenshot_resultat.png)
+
+
+# Steg 1: Vi starter fra toppen {.activity}
+
+Før vi starter må vi ha en helt enkel HTML-side. Siden vi har lært å bruke HTML5-tagger, så bruker vi det i denne oppgaven.
+
++ Last ned og pakk ut [layout.zip](ressurser/layout.zip)
++ Åpne `index.html` i din favoritt teksteditor
++ Legg nå til bildet `header.jpg` innenfor taggen `<header>` (husk å legge til `alt`-tekst)
+
+<toggle>
+  <strong>Hint</strong>
+  <hide>
+ 
+  ```html
+    <header>
+        <img src="header.jpg" alt="header">
+    </header>
+   ```
+  </hide>
+</toggle>
+
+
+Bildet er 1080 (bredde) × 250 (høyde) pixler så vi velger derfor å bruke bredden til bilde som et mål på hvor bred siden vår skal være. 
```

Comment:
Kanskje skrive "piksler ('pixels' på engelsk)" for å vise hvor "px"-forkortelsen kommer fra? (Det staves egentlig piksler på norsk)